### DISCUSSION TITLE
Calling prisma client from middleware

### DISCUSSION THREAD

**Discussion by puchm** [prisma/prisma#9802] [Q&A]
I am working on some middleware that in theory would need to query additional data from the database and apply that to the current query. Is there anything I need to consider when calling prisma from the middleware? How significant would the performance loss be?

**Comment by ryands17** [comment]
@puchm 👋 

The performance difference would be the same as the time taken to calling an additional query. There won't be any other loss in this case.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
